src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
+debugger;
 
-// write code here
+function getEmployees(list = document.getElementsByTagName('li')) {
```

Comment:
```suggestion
const list = document.querySelectorAll('li');

function getEmployees(list)) {
```
Доставай элементы вне функции, и обе функции используй с этим списком